app/eventyay/config/urls.py

```diff
@@ -60,6 +61,22 @@
 # Note: agenda and cfp patterns are now included under {organizer}/{event} in maindomain_urlconf.py
 # They are no longer at the root level
 
+# Auto-discover and load plugin URLs (similar to maindomain_urlconf.py)
```

Comment:
Your previous implementation is better. Because when we debug, we see a URL, we will want to trace back to the module which generates that URL.
A static registration like previous implementation helps us trace back easier (by searching for the string in the code base).

With this implementation, it is very hard to do so (there is no string to search).